Doc/using/cmdline.rst

```diff
@@ -834,7 +834,7 @@ conflict.
    order to force the interpreter to use ``ASCII`` instead of ``UTF-8`` for
    system interfaces.
 
-   Availability: \*nix
+   .. availability:: \*nix.
```

Comment:
Can we use "Unix" everywhere and not "*nix"?

Reply:
I didn't change any of the original text attached to Availability in order to just make this ticket about the new directive.  If you want, I can open another ticket to clean up things like this.

Reply:
> If you want, I can open another ticket to clean up things like this.

Yes, please don't do two things in the same PR. master also has this syntax.